src/resources/anim-clip.js

```diff
@@ -5,7 +5,6 @@ import { AnimData } from '../anim/evaluator/anim-data.js';
 import { AnimTrack } from '../anim/evaluator/anim-track.js';
 
 /**
- * @private
  * @class
```

Comment:
maybe this should stay private? There are no public properties of functions, so why the class.